### DISCUSSION TITLE
How do I put help into a cog?

### DISCUSSION THREAD

**Discussion by JavascriptLearner815** [Rapptz/discord.py#6371] [Q&A] [answered]
So I have a cog "Info" that I want the help command to be put into. So far, I have an invite command in the cog, but I don't know how to put the DEFAULT help command into it.

**Comment by Zomatree** [comment, marked as answer]
you can set `bot.help_command.cog` to the instance of the cog you would like to put the help command into.

**Reply by JavascriptLearner815** [reply]
So, like `bot.add_cog(Info(bot.help_command.cog))`?

<details>
<summary>Info Class</summary>

```python

class Info(commands.Cog):
    """Commands that have additional content or tips/tricks other than help"""

    @commands.command(description='Add --snip-- to another server')
    async def invite(self, ctx):
        """Add --snip-- to another server"""
        await ctx.send(invite_link)

```

</details>

**Reply by Zomatree** [reply]
No. inside the cogs `__init__`, set `bot.help_command.cog` the instance of the class.

**Reply by JavascriptLearner815** [reply]
```python

def __init__(self):
        bot.help_command.cog = self

```

?

**Reply by JavascriptLearner815** [reply]
Worked
